### DISCUSSION TITLE
suffixAction() for more form field types.

### DISCUSSION THREAD

**Discussion by dave-mills** [filamentphp/filament#5622] [Ideas]
Hi,

I've been exploring adding custom actions to form fields. The `suffixAction()` (and `prefixAction()`) works great for `TextInput`,  `Select` and `ColorPicker` fields. I think it would be great to also have it for other fields. In my case specifically, It'd be really useful on: 
- TextArea
- Checkbox
- CheckboxList 
- Radio
- Repeater (maybe?)

I can see that maybe not all of the attributes related to the `HasAffixes` trait would make sense on all fields (e.g., adding a `suffixIcon()` to a Repeater field), and probably the ability to do this for Text and Select fields covers 90% of all use cases, but given I've just bumped up against this I thought I'd bring up the idea!

**Comment by danharrin** [comment]
As long as it doesn't look weird styling wise, feel free to PR

**Comment by prykris** [comment]
I'm interested in seeing this developed, unable to find time for this myself.
